handler/src/main/java/io/netty/handler/stream/ChunkedStream.java

```diff
@@ -99,8 +100,14 @@ public void close() throws Exception {
         in.close();
     }
 
+    @Deprecated
```

Comment:
Do we even need this on the impl ? I mean shouldn't it be enough to have it on the interface ? Same question is true for other impls :)